techniques/html/H44.html

```diff
@@ -51,69 +51,66 @@
 
 <div class="default-grid with-gap leftcol">
 <aside class="box nav-hack sidebar standalone-resource__sidebar">
-    <header class="box-h">Page Contents</header>
+    <header class="box-h">このページの内容</header>
     <div class="box-i">
       <nav aria-label="page contents" class="navtoc">
-        <ul><li><a href="#technique">About this Technique</a></li>
-<li><a href="#description">Description</a></li>
-<li><a href="#examples">Examples</a></li>
-<li><a href="#resources">Related Resources</a></li>
-<li><a href="#related">Related Techniques</a></li>
-<li><a href="#tests">Tests</a></li>
+        <ul><li><a href="#technique">このテクニックについて</a></li>
+<li><a href="#description">解説</a></li>
+<li><a href="#examples">事例</a></li>
+<li><a href="#resources">関連リソース</a></li>
+<li><a href="#related">関連テクニック</a></li>
+<li><a href="#tests">検証</a></li>
 </ul>
       </nav>
     </div>
 </aside>
 
 <main id="main" class="standalone-resource__main">
       
-<h1><span>Technique H44:</span>Using label elements to associate text labels with form controls</h1>
+<h1><span>テクニック H44:</span>テキストラベルとフォームコントロールを関連付けるために、label 要素を使用する</h1>
 
 
 
 
 <section id="technique" class="box">
-	<h2 class="box-h box-h-icon">About this Technique</h2>
+	<h2 class="box-h box-h-icon">このテクニックについて</h2>
 	<div class="box-i">
 
 
   
-  <p>This technique relates to:</p>
+  <p>このテクニックは、次に関連する:</p>
   <ul>
   
     <li>
       
-<a href="https://w3c.github.io/wcag/understanding/non-text-content">1.1.1: Non-text Content</a>
-(<strong>Sufficient</strong>)
+<a href="https://waic.jp/translations/WCAG22/Understanding/non-text-content">1.1.1: 非テキストコンテンツ</a>
+(<strong>十分なテクニック</strong>)
     </li>
   
     <li>
       
-<a href="https://w3c.github.io/wcag/understanding/info-and-relationships">1.3.1: Info and Relationships</a>
-(<strong>Sufficient</strong>
-  when used for making information and relationships conveyed through presentation programmatically
-                     determinable)
+<a href="https://waic.jp/translations/WCAG22/Understanding/info-and-relationships">1.3.1: 情報及び関係性</a>
+(提示を通じて伝えられる情報及び関係性をプログラムによる決定が可能にするために使用される場合に<strong>十分なテクニック</strong>)
```

Comment:
programmatically determinable を「プログラムによる解釈が可能」としました。
```suggestion
(提示を通じて伝えられる情報及び関係性をプログラムによる解釈が可能にするために使用される場合に<strong>十分なテクニック</strong>)
```